images/oc-build-deploy-dind/openshift-templates/elasticsearch/pvc.yml

```diff
@@ -34,6 +34,12 @@ parameters:
   - name: DEPLOYMENT_STRATEGY
     description: Strategy of Deploymentconfig
     value: "Rolling"
+  - name: PERSISTENT_STORAGE_CLASS
+    description: Name of the Storage Class to use
+    value: "bulk"
```

Comment:
let's use the default value `""` as `bulk` is usually some kind of NFS filesystem on our clusters which we don't want :)